### PR TITLE
DM-35744: Add some additional trace logging to RelationGraph

### PR DESCRIPTION
**Pull request by fritzm**

### CODE REVIEW

File: src/qana/RelationGraph.cc [hunk 1]
```diff
@@ -878,6 +882,27 @@ void computeMinimumOverlap(Vertex& vtx) {
...
                 u->overlap = requiredOverlap;
                 q.enqueue(u);
+                LOGS(_log, LOG_LVL_TRACE, " (updated and queued)");
+            }
+        }
+    }
+}
+
+/// dump graph
+void dumpGraph(std::list<Vertex> const& vertices) {
```
Comment (iagaponenko) [inline src/qana/RelationGraph.cc:892]: Shouldn't this free-standing function be put into the anonymous namespace at the beginning of this compilation unit?

Comment (iagaponenko) [reply src/qana/RelationGraph.cc:892]: I don't see any harm to have it in the `lsst::qserv::qana` namespace. I'm just curious what's the right code design rule here.

**Review by iagaponenko — APPROVED** [review]
Looks good to me

File: src/qana/RelationGraph.cc (hunk 1 above)
Comment (fritzm) [reply src/qana/RelationGraph.cc:892]: Oh, I though I *had* put in in the anon namespace...  Will fix, thx

**Pull request merged**